sdk/tables/azure-data-tables/samples/async_samples/sample_batching_async.py

```diff
@@ -77,13 +64,15 @@ async def sample_transaction(self):
         except ResourceExistsError:
             print("Table already exists")
 
-        await self._create_entities()
+        await self.table_client.upsert_entity(entity2)
+        await self.table_client.upsert_entity(entity3)
+        await self.table_client.upsert_entity(entity4)
 
         operations: List[TransactionOperationType] = [
-            ("create", self.entity1),
-            ("delete", self.entity2),
-            ("upsert", self.entity3),
-            ("update", self.entity4, {"mode": "replace"}),
+            ("upsert", entity1),
```

Comment:
`"upsert"` is already used on line 74, so I think this should be `"create"`?

Reply:
Updated